scipy/linalg/tests/test_cython_lapack.py

```diff
@@ -14,4 +14,9 @@ def test_dlamch(self):
         for c in [b'e', b's', b'b', b'p', b'n', b'r', b'm', b'u', b'l', b'o']:
             assert_allclose(cython_lapack._test_dlamch(c),
                             lapack.dlamch(c))
-
+        
+    def test_complex_ladiv(self):
```

Comment:
Looks like some whitespace has crept in
```suggestion

    def test_complex_ladiv(self):
```